<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Module 2</title>
  <link rel="shortcut icon" href="images/favicon.ico"/>

  <!-- Javascript -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
  <script src="js/jquery-1.11.0.min.js"></script>
  <script src="js/lightbox.min.js"></script>

  <!--CSS Reset -->
  <!--<link href="http://2014.igem.org/Team:Tec-Monterrey/ITESM14_reset.css?action=raw&ctype=text/css" type="text/css" rel="stylesheet"> -->

  <!-- Bootstrap CSS -->
  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css?action=raw&ctype=text/css" type="text/css" rel="stylesheet">
  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">

  <!--Custom CSS -->
  <link href="css/ITESM14_main.css" type="text/css" rel="stylesheet">
  <link href="css/ITESM14_navbar.css" type="text/css" rel="stylesheet">
  <link href="css/lightbox.css" rel="stylesheet" />
  <link href='http://fonts.googleapis.com/css?family=Yanone+Kaffeesatz:400,200,300,700&subset=latin,latin-ext' rel='stylesheet' type='text/css'>

  <script> 
    $(function(){
      $("#wiki-header").load("header.html"); 
    });
  </script> 
  <script> 
    $(function(){
      $("#wiki-footer").load("footer.html"); 
    });
  </script>
</head>

<div id="wiki-body">

<body>

  <header id="wiki-header" class="initial-nav-scroll-up">
  </header>

<div id ="wiki-content" class="main-content">

  <div class="jumbotron col-md-offset-3" style="background: transparent; padding-top: 0; margin-bottom: 0; padding-bottom: 0; padding-left:0;">
    <div class="container">
		<h1 style="color: #f4ff3b; font-size: 500%; padding: 0; margin: 50; font-weight: lighter;">Module 2: Phage engineering </h1>    
	</div>
  </div>

  <div class="well">
  <div class="container">

    <h1>Aim</h1>
      <section id="column"><p> 
	  This module is about two modifications made to the phage genome using PCR. The first change is the addition of a peptide to the PIII protein to modify the tropism of the phage, so it will be able to introduce itself into the cancerous cells. The other modification consisted of adding a PluX promoter before PII, this will allow us to express the phage only when bacteria reach a certain cell density.
      </p></section>
    <hr>

    <h1>Background</h1>
		<h2> About M13 Bacteriophage</h2>
			<section id="column">
				<p>
				Bacteriophages are a class of extremely specific viruses that infect bacteria and lack the ability to infect more complex organisms because of major differences in cell-surface molecules and in the intracellular machinery that is key for their replication [3]. To introduce themselves into the bacteria they attach to specific receptors on the surface of the host cell, the specificity of these interactions influences the bacterial host range [6].
				</p>
				<p>
				M13 is an <em>Escherichia coli-specific</em> filamentous bacteriophage that consists of a single-stranded DNA core surrounded by a proteinaceous coat [4]. The genome of this virus contains 11 genes with 5 encoding the coat proteins (P3, P6, P7, P8 and P9) and the remaining being necessary for viral replication and assembly [2].The infection is initiated when P3 binds to the F pilus on the surface of an <em>E. coli</em> cell which facilitates the translocation of the viral DNA into the cytoplasm. Inside the host cell, the single stranded DNA is converted to a double-stranded form by P2. This form is further duplicate and used as the template for the production of all the phage proteins. Also it is used as the template for the production of new single-stranded viral DNA through a “rolling circle” replication mode. Phage assembly is initiated by the incorporation of P7 and P9 at one end of the particle. The elongation process then follows in which thousands of P8 molecules are assembled along the length of the particle. The process is terminated by the addition of P6 and P3 at the other end of the particle [2].  
				</p>
				<p>
				The potential use of the bacteriophages includes therapy for bacterial infections, modifying the microenvironment of tumors for anticancer treatment and serving as a platform for foreign peptides that may induce anticancer <strong>effects</strong>[3]. One of the first recorded uses of bacteriophages in therapy was the successful treatment of dysentery in 1919 in which four different patients were treated with a phage preparation. Likewise in 1921 the use of these phages was found to be effective in the treatment of staphylococcal skin disease [7]. Another use of bacteriophages is the treatment of tumors. It was already reported by mid-1800s that tumor regression can occur during natural viral infections. For example some viruses are known to be oncolytic, this means that their propagations in cancer cells is much more efficient than in normal cells and others like adenoviruses can replicate effectively if the p53 is inactivated, which happens in at least 50% of human cancers [8].
				</p>
				<p>
				Yet the use of the oncolytic viruses presents a danger in itself. One of the methods to circumvent these dangers is to use bacteriophages which have been reported since 1940 to have antitumor activity in mice and rabbits [9]. By using these viruses the excretion of cytokines can be expressed and through this the immunosuppressive tumor microenvironment can be altered to promote tumor toxicity. At the same time, phage display can be used to increase specificity between the tumors and the bacteriophages [10]. 
				</p>
			</section>
			
		<h2> Phage display and phage engineering </h2>
			<section id="column">
				<p>
				Phage display is a technique in phage engineering and a useful method for the selection and engineering of polypeptides with desired binding specificities [1]. This method consists on fusing polypeptide libraries to phage coat proteins, which results in the phage proteins displaying the polypeptides on their surfaces [2]. Phage display can also be used to make a cell permeable to the bacteriophages by modifying the coat proteins and further engineering can be done so that the viruses can release drugs once inside the cell through either a covalent bond conjugation between the virus and drug or through the modification of its genome to make it carry a desired gene [11]. 
				</p>
				<p>
				To do this a hybrid phage is created in which the displayed fusion protein is included as an additional component of a phage coat that contains wild-type copies of all coat proteins. This can be done by using a phagemid, a plasmid which contains a replication origin from the virus that will allow it to produce single stranded DNA.
				</p>
				<p>
				-----
				</p>
				<p>
				DNA sequences that allow packaging into phage particles [2?].This plasmid replicated in E. coli as a double stranded plasmid, but coinfection with an assisted phage results in the production of single-stranded phagemid DNA which is then packaged into phage particles. For phage display, the phagemid contains a gene that encodes an additional copy of one of the coat proteins fused to the polypeptide that is to be displayed, the resulting phage contains both wild-type coat proteins from the helper phage and also the fusion coat protein from the phagemid [2]. In the project at hand a modification to P3 will be made in order to introduce the Pep42 protein in it. Pep42 is a cyclic 13-mer oligopeptide, CTVALPGGYVRVC, that specifically binds to glucose-regulated protein 78 (GRP78) [?]. GRP78 is an intracellular chaperone and member of the heat shock protein 70 (HSP70) family that provides a protective cellular response against stress conditions. Normal GRP78 expression is maintained at low levels but is upregulated in stress environments and induced in tumor environments. GRP78 overexpression has been demonstrated in a variety of tumors, such as prostate, colon, skin, and breast cancers. This GRP78 overexpression represents a potentially attractive target as it results in the presence of GRP78 molecules on the cell surface of these cancer cells [?]. The bacteriophage´s internalization will then be mediated by the pep42-GRP78 ligand made possible by the addition of the Pep42 in the N-terminal of P3, the phagemid on the other hand will have the DNA of the apoptin and the survivin siRNA which give the phage its anti-cancer properties.
				</p>
				<p>
				This project’s focus is using this method on the M13 bacteriophage in order to express the peptide pep42, which is a cell-penetrating peptide that will allow us to use the phages as a vehicle for site-directed cancer therapy.
				</p>
			</section>
	<hr>
	
	<h1> PCR mutagenesis</h1>
		<section id="column">
			<h2> Phage genome VCSM13 </h2>
				<p>
				It’s genome has 8,669 bp which is longer than regular phages. It is noteworthy that this phage has a mutation in the oriF1, which is the site where the PII protein makes a cut and starts the rolling circle replication and the generation of single-stranded DNA (ssDNA), a process that will be interrupted by the mutation. However, the phage is still able to replicate because it has a bacterial origin of replication (P15A). As the phage cannot generate ssDNA, it will allow us to use with a phagemid, whose DNA will be packed in the phage.
				</p>
				<figure>
				<a href="images/module2/VCSM13.jpg" data-lightbox="VCSM13" data-title="Fig. 1 Phage genome VCSM13"><img style="margin:0px auto;display:block" src="images/module2/VCSM13.jpg"></a>
				<center><figcaption> Fig. 1 Phage genome VCSM13 </Figcaption></center>
				</figure>
			<h2>PCR of peptide PIII </h2>
				<p>
				In order to change the tropism of the virus, we performed a PCR to insert the pep42 peptide to the phage pIII. This peptide will allow the phage to internalize into cancerous cells when it binds to the GRP78 receptor. Two pairs of primers were designed to contain the sequence encoding the peptide sequence; also, the restriction site KpnI was added in order to be subsequently re-ligated.
				</p>
				<p>
				These are the primers that were used in this section:
				</p>
				<p><ul id="primers">
					<li> <strong>F1:</strong> GTG GTA CCT GTA CCG TTG CGC TGC CGG GCG GTT ATG TGC GTG TTT GTG CTG AAA CTG TTG AAA GTT G </li>
					<li> <strong>R1:</strong> GTG GTA CCG GAG TGA GAA TAG AAA GGA AC </li>
					<li> <strong>F2:</strong> GTG GTA CCG CTG AAA CTG TTG AAA GTT G </li>
					<li> <strong>R2:</strong> GTG GTA CCA CAA ACA CGC ACA TAA CCG CCC GGC AGC GCA ACG GTA CAG GAG TGA GAA TAG AAA GGA AC </li>
				</ul></p>
				<figure>
				<a href="images/module2/PIII.jpg" data-lightbox="PIII" data-title="Fig. 2 PCR of peptide PIII"><img style="margin:0px auto;display:block" src="images/module2/PIII.jpg"></a>
				<center><figcaption> Fig. 2 PCR of peptide PIII </Figcaption></center>
				</figure>
			<h2> PCR of PLux </h2>
				<p>
				PCR will be used as well to insert will promoter for the quorum sensing system (PLux) before the gene coding for PII. As bacteria are only able to colonize the tumour area, the cell density required to activate the pLux will be achieved when the bacteria are concentrated there.
				</p>
				<p>
				The PCR employed two pairs of primers containing the sequence encoding the promoter, as well as the sequence of restriction sites XbaI and SpeI to subsequently re-link and be put on the pSB1C3 backbone to the match the iGEM BioBrick <span id="blue">(BBa….)</span>.
				</p>
				<p><ul id="primers">
					<li><strong>F1:</strong> GTT CTA GAA CCT GTA GGA TCG TAC AGG TTT ACG CAA GAA AAT GGT TTG TTA TAG TCG AAT AAA CAA TCT TCC TGT TTT TGG </li>
					<li><strong>R1:</strong> GTA CTA GTG CCG AAA TCG GCA AAA TC </li>
					<li><strong>F2:</strong> GTT CTA GAA CCT GTA GGA TCG TAC AGG TTT ACG CAA GAA AAT GGT TTG TTA TAG TCG AAT AAA GCT TTT CTG ATT ATC AAC CGG </li>
					<li> <strong>R2:</strong> GTA CTA GTC GAA ATC GGC AAA ATC CCT </li>
				</ul></p>
				<figure>
				<a href="images/module2/PLux.jpg" data-lightbox="PLux" data-title="Fig. 3 PCR of PLux"><img style="margin:0px auto;display:block" src="images/module2/PLux.jpg"></a>
				<center><figcaption> Fig. 3 PCR of PLux </Figcaption></center>
				</figure>
			<h2> pET28a Phagemid </h2>
				<p>
				The phagemid has 5,369 bp and has an origin of replication F1, so it is able to generate ssDNA. In addition, it has the lacZ gene in the MCS, which uses to select the colonies that were transformed with the phagemid linked to the effector.
				</p>
				<figure>
				<a href="images/module2/Phagemid.jpg" data-lightbox="Phagemid" data-title="Fig. 4 pET28a Phagemid"><img style="margin:0px auto;display:block" src="images/module2/Phagemid.jpg"></a>
				<center><figcaption> Fig. 4 pET28a Phagemid </Figcaption></center>
				</figure>
		</section>
	<hr>
	
	<h1>Results</h1>
	  <p>
	  *
      </p>
	<hr>
	
	<h1>References</h1>
	  <ol>
		<li><p>
		*
		</p></li>
	  </ol>
	<hr>
	
  </div> 
  </div>
</div>

</body>

<footer id="wiki-footer" >
</footer>

<script src="js/ITESM14_main.js"></script>

<script>
  $( "#project" ).addClass("active");
</script>

</div>

</html>

